pennylane/logging/configuration.py

```diff
@@ -99,3 +104,40 @@ def config_path():
     """
     path = os.path.join(_path, "log_config.toml")
     return path
+
+
+def show_system_config():
+    """
+    This function opens the logging configuration file in the system-default browser.
+    """
+    # pylint:disable = import-outside-toplevel
+    import webbrowser
```

Comment:
Given that this function isn't needed in most cases, I'd rather keep it at this level to avoid the inherent module level lookup, since those can slow down for scaled out workloads at startup.